TypeGen: protocol.Attestation not mapped to TypeScript type

## Problem

TypeScript typecheck fails with:

```
types/generated/typescript/server.ts(390,17): error TS2304: Cannot find name 'Attestation'.
```

## Root Cause

`server/prompt_handlers.go:90`:
```go
type PromptDirectResponse struct {
    Attestation *protocol.Attestation `json:"attestation,omitempty"`
    ...
}
```

TypeGen outputs:
```ts
export interface PromptDirectResponse {
    attestation?: Attestation | null;  // ❌ Attestation not imported
    ...
}
```

But `Attestation` is neither imported nor defined.

## Expected Behavior

**Option 1:** Map `protocol.Attestation` → `As` (from './types')
```ts
import { As } from './types';

export interface PromptDirectResponse {
    attestation?: As | null;
    ...
}
```

**Option 2:** Import proto-generated `Attestation`
```ts
import { Attestation } from '../web/ts/generated/proto/plugin/grpc/protocol/atsstore';
```

## Context

- `protocol.Attestation` is a protobuf type from `plugin/grpc/protocol/atsstore.proto`
- TypeScript has `As` interface in `types/generated/typescript/types.ts` 
- TypeGen should know how to map protobuf types to TypeScript equivalents

## Reproduce

```bash
bun run typecheck
```

## Workaround

None currently - blocks TypeScript builds.

--- a/typegen/typescript/generator.go
+++ b/typegen/typescript/generator.go
@@ -96,6 +96,8 @@ var TypeMapping = map[string]string{
 	"NullString":     "string | null",
 	"NullInt64":      "number | null",
 	"NullTime":       "string | null",
+	// Cross-package struct references
+	"protocol.Attestation": "As",
 }
 
 // typeConverterConfig is the TypeScript-specific type conversion configuration

--- a/types/generated/typescript/server.ts
+++ b/types/generated/typescript/server.ts
@@ -387,7 +387,7 @@ export interface PromptDirectResponse {
   /**
    * Full attestation with signature
    */
-  attestation?: Attestation | null;
+  attestation?: As | null;
   prompt_tokens?: number;
   completion_tokens?: number;
   total_tokens?: number;

